<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TvShowWidgetEpisode</class>
 <widget class="QWidget" name="TvShowWidgetEpisode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>695</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>21</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QToolButton" name="buttonRevert">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="toolTip">
        <string>Episode has changed. Click to revert changes.</string>
       </property>
       <property name="styleSheet">
        <string notr="true">border: none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="MediaElch.qrc">
         <normaloff>:/img/arrow_circle_left.png</normaloff>:/img/arrow_circle_left.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="episodeName">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Episode Title</string>
       </property>
       <property name="indent">
        <number>4</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="styleSheet">
      <string notr="true">QGroupBox {
	border: 1px solid rgba(0, 0, 0, 100);
	border-radius: 5px;
	background-color: rgba(255, 255, 255, 200);
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab1">
         <attribute name="title">
          <string>Information</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QFormLayout" name="formLayout">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::ExpandingFieldsGrow</enum>
            </property>
            <property name="rowWrapPolicy">
             <enum>QFormLayout::WrapLongRows</enum>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Files</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="files">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Name</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="name"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Show Title</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="showTitle"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Season</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QSpinBox" name="season"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>Episode</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QSpinBox" name="episode"/>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Rating</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QDoubleSpinBox" name="rating">
              <property name="decimals">
               <number>1</number>
              </property>
              <property name="maximum">
               <double>10.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Certification</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QComboBox" name="certification">
              <property name="editable">
               <bool>true</bool>
              </property>
              <property name="sizeAdjustPolicy">
               <enum>QComboBox::AdjustToContents</enum>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>First Aired</string>
              </property>
             </widget>
            </item>
            <item row="7" column="1">
             <widget class="QDateEdit" name="firstAired">
              <property name="displayFormat">
               <string>dd.MM.yyyy</string>
              </property>
             </widget>
            </item>
            <item row="8" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Play Count</string>
              </property>
             </widget>
            </item>
            <item row="9" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Last Played</string>
              </property>
             </widget>
            </item>
            <item row="9" column="1">
             <widget class="QDateTimeEdit" name="lastPlayed">
              <property name="displayFormat">
               <string>dd.MM.yyyy HH:mm</string>
              </property>
             </widget>
            </item>
            <item row="10" column="0">
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Studio</string>
              </property>
             </widget>
            </item>
            <item row="10" column="1">
             <widget class="QLineEdit" name="studio"/>
            </item>
            <item row="11" column="0">
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>Overview</string>
              </property>
             </widget>
            </item>
            <item row="11" column="1">
             <widget class="QTextEdit" name="overview"/>
            </item>
            <item row="8" column="1">
             <widget class="QSpinBox" name="playCount"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Extended</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QFormLayout" name="formLayout_2">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::ExpandingFieldsGrow</enum>
            </property>
            <property name="rowWrapPolicy">
             <enum>QFormLayout::WrapLongRows</enum>
            </property>
            <item row="1" column="0">
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>Directors</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,0,0">
              <item>
               <widget class="QTableWidget" name="directors">
                <property name="maximumSize">
                 <size>
                  <width>300</width>
                  <height>100</height>
                 </size>
                </property>
                <property name="editTriggers">
                 <set>QAbstractItemView::DoubleClicked</set>
                </property>
                <property name="showGrid">
                 <bool>false</bool>
                </property>
                <attribute name="horizontalHeaderVisible">
                 <bool>false</bool>
                </attribute>
                <attribute name="horizontalHeaderStretchLastSection">
                 <bool>true</bool>
                </attribute>
                <attribute name="verticalHeaderVisible">
                 <bool>false</bool>
                </attribute>
                <column>
                 <property name="text">
                  <string>Studio</string>
                 </property>
                </column>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <item>
                 <widget class="QPushButton" name="buttonAddDirector">
                  <property name="toolTip">
                   <string>Add Director</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="MediaElch.qrc">
                    <normaloff>:/img/plus.png</normaloff>:/img/plus.png</iconset>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="buttonRemoveDirector">
                  <property name="toolTip">
                   <string>Remove Director</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="MediaElch.qrc">
                    <normaloff>:/img/minus.png</normaloff>:/img/minus.png</iconset>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>Writers</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,0,0">
              <item>
               <widget class="QTableWidget" name="writers">
                <property name="maximumSize">
                 <size>
                  <width>300</width>
                  <height>100</height>
                 </size>
                </property>
                <property name="editTriggers">
                 <set>QAbstractItemView::DoubleClicked</set>
                </property>
                <property name="showGrid">
                 <bool>false</bool>
                </property>
                <attribute name="horizontalHeaderVisible">
                 <bool>false</bool>
                </attribute>
                <attribute name="horizontalHeaderStretchLastSection">
                 <bool>true</bool>
                </attribute>
                <attribute name="verticalHeaderVisible">
                 <bool>false</bool>
                </attribute>
                <column>
                 <property name="text">
                  <string>Genre</string>
                 </property>
                </column>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QPushButton" name="buttonAddWriter">
                  <property name="toolTip">
                   <string>Add Writer</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="MediaElch.qrc">
                    <normaloff>:/img/plus.png</normaloff>:/img/plus.png</iconset>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="buttonRemoveWriter">
                  <property name="toolTip">
                   <string>Remove Writer</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="MediaElch.qrc">
                    <normaloff>:/img/minus.png</normaloff>:/img/minus.png</iconset>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_14">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Thumbnail</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MyLabel" name="thumbnail">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>112</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>112</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>Click to Change</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="MediaElch.qrc">:/img/pictures_alt.png</pixmap>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QToolButton" name="buttonPreviewBackdrop">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="toolTip">
             <string>Full Preview</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QToolButton {
	border: none;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="MediaElch.qrc">
              <normaloff>:/img/zoom_in.png</normaloff>:/img/zoom_in.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="thumbnailResolution">
            <property name="toolTip">
             <string>Resolution</string>
            </property>
            <property name="text">
             <string>Resolution</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLabel</class>
   <extends>QLabel</extends>
   <header>smallWidgets/MyLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MediaElch.qrc"/>
 </resources>
 <connections/>
</ui>
